packages/drivers/debugger/api-extractor/api-extractor.legacy.json

```
@@ -1,4 +1,8 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-report.esm.legacy.json"
+	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-report.esm.legacy.json",
+	"apiReport": {
+		"enabled": true,
+		"reportVariants": ["alpha", "beta"]
```

This overrides the base config which currently generates alpha and public, so this change will result in the legacy public report no longer being generated. I would recommend doing what I did in the client-utils PR and replace the alpha report with the beta report.

(We're also omitting the "enabled" property to avoid introducing too many overrides. The linter will warn, but the tooling doesn't actually care).

```suggestion
		"reportVariants": ["public", "beta"]
```

---

That's fine too